frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Edit_HPA_Action.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Edit HPA action
+	As a user, I want to remove the HPA assigned to a workload
```

Comment:
Feature is to edit the HPA and not removing it. Can you please update the feature description?